sw/device/tests/sim_dv/uart_tx_rx_test.c

```diff
@@ -432,6 +432,10 @@ static bool execute_test(const dif_uart_t *uart) {
   return true;
 }
 
+const test_config_t kTestConfig = {
+    .can_clobber_uart = true,
```

Comment:
Tests run in DV do not use UART for logging. This can be removed.

Reply:
I guess it should be `const test_config_t kTestConfig = {}` instead.